aerogear/data-sync-ui
# Reduce structure view

## Motivation

Reduce the structure view in the schema editor to what we discussed in the UX review: no interactivity, only list the types and their properties. I suggest that we write a different component for the structure view in the resolver tab because it has a different organization and more features.

## Verification Steps

Paste this Schema and verify the types are rendered correctly:

```
  type Meme {
    id: ID! @isUnique
    photoUrl: String!
  }
  
  type Query {
    allMemes:[Meme!]!
  }
  
  type Mutation {
    createMeme(photoUrl: String!):Meme!
  }
  
  type Subscription {
    _: Boolean
  }
```

## Checklist:


## Progress


## Screenshot

![bildschirmfoto 2018-07-26 um 18 36 46](https://user-images.githubusercontent.com/1851198/43275816-30f12300-9103-11e8-95fd-d9aca6571404.png)
